nestjs-gateway/README.md

```
@@ -0,0 +1,98 @@
+<p align="center">
+  <a href="http://nestjs.com/" target="blank"><img src="https://nestjs.com/img/logo-small.svg" width="120" alt="Nest Logo" /></a>
+</p>
+
+[circleci-image]: https://img.shields.io/circleci/build/github/nestjs/nest/master?token=abc123def456
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Unused link or image reference definition: "circleci-image"

(MD053, link-image-reference-definitions)